app/templates/my-projects/project/recommendations/add.hbs

```diff
@@ -10,61 +10,372 @@
+              @searchEnabled={{false}}
+              @options={{this.recOptions}}
+              @placeholder="Select a recommendation"
+              @selected={{this.dispositionForAllActions.recommendation}}
+              @onchange={{action (mut this.dispositionForAllActions.recommendation)}}
+              as |recOption|
+            >
+              {{recOption}}
+            </PowerSelect>
+            {{#if (or (eq participantType 'CB') (eq participantType 'BB'))}}
+              <legend>
+                <strong>
+                  Vote Counts
+                </strong>
+              </legend>
+              <div class="grid-x grid-margin-x">
+                <div  class="cell medium-3">
+                  <label for="all-actions-votes-in-favor">
+                    Votes In Favor
+                  </label>
+                  <Input  
+                    @id="all-actions-votes-in-favor"
+                    @type="number"
+                    @value={{this.dispositionForAllActions.votinginfavorrecommendation}}
+                    @key-up={{fn (mut this.dispositionForAllActions.votinginfavorrecommendation) value="target.value"}}
+                  />
+                </div>
+                <div  class="cell medium-3">
+                  <label for="all-actions-votes-against">
+                    Votes Against
+                  </label>
+                  <Input
+                    @id="all-actions-votes-against"
+                    @type="number"
+                    @value={{this.dispositionForAllActions.votingagainstrecommendation}}
+                    @key-up={{fn (mut this.dispositionForAllActions.votingagainstrecommendation) value="target.value"}}
+                  />
+                </div>
+                <div  class="cell medium-3">
+                  <label for="all-actions-votes-abstain">
+                    Abstain
+                  </label>
+                  <Input
+                    @id="all-actions-votes-abstain"
+                    @type="number"
+                    @value={{this.dispositionForAllActions.votingabstainingonrecommendation}}
+                    @key-up={{fn (mut this.dispositionForAllActions.votingabstainingonrecommendation) value="target.value"}}
+                  />
+                </div>
+                <div  class="cell medium-3">
+                  <label for="all-actions-votes-total-members">
+                    Total Members
+                  </label>
+                  <Input
+                    @id="all-actions-votes-total-members"
+                    @type="number"
+                    @value={{this.dispositionForAllActions.totalmembersappointedtotheboard}}
+                    @key-up={{fn (mut this.dispositionForAllActions.totalmembersappointedtotheboard) value="target.value"}}
+                  />
+                </div>
+              </div>
+            {{/if}}
+          </fieldset>
+        </div>
+      {{/if}}
+      {{#if (eq this.allActions false)}}
```

Comment:
Wait... `allActions` is a boolean, no? So this could just be `{{#if this.allActions}}`, right?

Reply:
I wanted to make it a strict equals to `false`, because `this.allActions` starts out as `undefined`. 
I wanted to only show the "By Action" Recommendation forms after the user has selected either Yes or No.

Reply:
It's `null` by default, right? So we explicitly want to know if it's `true` or `false`.